aospy/region.py

```diff
@@ -42,20 +44,35 @@ def mask_var(self, data):
 
     @staticmethod
     def _get_land_mask(data, do_land_mask):
+        if not do_land_mask:
+            return 1
+        try:
+            land_mask = data.land_mask.copy()
+        except AttributeError:
+            # TODO: Implement aospy built-in land mask to default to.
```

Comment:
Yes, that's pretty much what I had in mind.  I'll open a separate issue for this.